packages/block-editor/src/components/writing-flow/index.js

```diff
@@ -364,7 +364,7 @@ class WritingFlow extends Component {
 					aria-hidden
 					tabIndex={ -1 }
 					onClick={ this.focusLastTextField }
-					className="wp-block editor-writing-flow__click-redirect block-editor-writing-flow__click-redirect"
```

Comment:
What's the reason for this change?

I seem to recall some need for this to imitate `BlockListBlock`:

https://github.com/WordPress/gutenberg/blob/916cbb0817f3c60b54067b361f56475acf511cee/packages/block-editor/src/components/block-list/block.js#L395-L398

Reply:
Not sure if it makes much difference. Without the class, clicks anywhere under the block list will redirect. With the class, only clicks vertically under it.

Reply:
Doesn't the old behaviour make more sense?